api/datasets_api.go

```diff
@@ -20,26 +20,8 @@ import (
 
 const (
 	datasetDocType         = "dataset"
-	editionDocType         = "edition"
-	versionDocType         = "version"
-	downloadServiceToken   = "X-Download-Service-Token"
 	dimensionDocType       = "dimension"
```

Comment:
I would consider pulling the dimension and metadata stuff out as well, since it's found at a different endpoint. some possible options:
all into the version file
create a dimensions.go for all the dimension related stuff, and put metadata in version file
create a dimensions.go and metadata.go